linode/helper/version.go

```diff
@@ -0,0 +1,31 @@
+package helper
+
+import (
+	"fmt"
+	"runtime/debug"
+)
+
+func getDepVersion(depPath string) string {
+	info, ok := debug.ReadBuildInfo()
+
+	if !ok {
+		fmt.Println("No build information available")
+		return ""
+	}
+
+	for _, dep := range info.Deps {
+		if dep.Path == depPath {
+			return dep.Version
+		}
+	}
+
+	return ""
+}
+
+func GetSDKv2Version() string {
+	return getDepVersion("github.com/hashicorp/terraform-plugin-sdk/v2")
+}
+
+func GetFrameworkVersion() string {
```

Comment:
@zliang-akamai Could the Framework UA also be updated to consume this function? 

https://github.com/linode/terraform-provider-linode/blob/dev/linode/framework_provider_config.go#L293